dask_expr/expr.py

```diff
@@ -1068,6 +1084,7 @@ def _task(self, index: int):
 
 
 class Map(Elemwise):
+    projection_passthrough = True
```

Comment:
This one seems suspect to me

Reply:
I just checked and saw that there is no `DataFrame.map` so I'll retract this comment.

Reply:
map is applied to every value, there isn't much you can do with regards to structure